### DISCUSSION TITLE
403 on deleteme API

### DISCUSSION THREAD

**Discussion by dayeggpi** [umputun/remark42#1402] [General]
Hi team,

Whenever I click on "request my data removal" I get the following error in Chrome...
![sshots03216](https://user-images.githubusercontent.com/11727370/176899119-e897868b-5134-4f5f-ad47-b7ec745c2f7d.png)

and get a 403 error on https://mydomain/api/v1/deleteme?site=mysite

Could you please guide me to what did I miss ?

Below my env :

>    environment:
      - REMARK_URL=https://mydomain
      - SECRET=secret
      - SITE=mysite
      - PROXY_CORS=false
      - DEBUG=true
      - AUTH_SAME_SITE=none
      - AUTH_ANON=true
      - ALLOWED_HOSTS=mydomain

**Comment by paskal** [comment]
I see that it doesn't work for anonymous users, giving 403. @akellbl4 should we maybe remove the button from the anonymous user profile?

**Reply by akellbl4** [reply]
Seems reasonable

**Reply by paskal** [reply]
I created https://github.com/umputun/remark42/issues/1460 about it.
